File: web/src/app/modules/roosid/model/round.ts
import {Answer} from "./answer";

export class Round {
  id: number = undefined;
  question: string = undefined;
  answers: Answer[] = undefined;
  multiplayer: number = undefined;
  score: number = undefined;
  scoreToWin: number = undefined;
  winnerTeamId: number = undefined;
  answeringTeamId: number = undefined;
  active: boolean = undefined;
}

File: core/src/main/java/eu/kotkas/villak/core/roosid/model/Round.java
package eu.kotkas.villak.core.roosid.model;

import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Round implements Serializable {
    private static final long serialVersionUID = 2320362160409094472L;
    private long id;
    private String question;
    private List<Answer> answers;
    private long multiplayer;
    private long score;
    private long scoreToWin;
    private Long winnerTeamId;
    private Long answeringTeamId;
    private boolean active;
}
